spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -555,385 +540,264 @@ If the @WebService.targetNamespace annotation is on a service
-
 The service implementation bean MAY implement the service endpoint
 interface, but is not REQUIRED to do so.
 
 This member-value is not allowed on endpoint interfaces.
-
-a|
-None.
+|None.
 
 The Web Service contract is generated from annotations on the service
 implementation bean. If a service endpoint interface is required by the
 target environment, it will be generated into an implementation-defined
 package with an implementation-defined name.
-
 |===
 
-== 4.1.2 Annotation Type Definition
+==== Annotation Type Definition
 
+[source,java]
+----
 @Retention(value=RetentionPolicy.RUNTIME)
-
-@Target(\{TYPE})
-
-public @interface WebService \{
-
-String name() default "";
-
-String targetNamespace() default "";
-
-String serviceName() default "";
-
-String portName() default "";
-
-String wsdlLocation() default "";
-
-String endpointInterface() default "";
-
+@Target({TYPE})
+public @interface WebService {
+  String name() default "";
+  String targetNamespace() default "";
+  String serviceName() default "";
+  String portName() default "";
+  String wsdlLocation() default "";
+  String endpointInterface() default "";
 };
+----
 
-*4.1.3 Example +
-Java source:*
+==== Example
+*Java source:*
 
+[source,java]
+----
 /**
-
 * Annotated Implementation Object
-
 */
-
 @WebService(
-
-name = "EchoService",
-
-targetNamespace =
-"http://www.openuri.org/2004/04/HelloWorld[[.underline]#http://www.openuri.org/2004/04/HelloWorld#]"
-
+  name = "EchoService",
+  targetNamespace = "http://www.openuri.org/2004/04/HelloWorld"
 )
-
-public class EchoServiceImpl \{
-
-@WebMethod
-
-public String echo(String input) \{
-
-return input;
-
+public class EchoServiceImpl {
+   @WebMethod
+   public String echo(String input) {
+      return input;
+   }
 }
+----
 
-}
+=== Annotation: javax.jws.WebMethod
 
-*__4.2 Annotation: javax.jws.WebMethod +
-__4.2.1 Description*
+==== Description
 
 Customizes a method that is exposed as a Web Service operation. The
 WebMethod annotation includes the following member-value pairs:
 
-[cols=",,",]
+[cols=3, options=header]
 |===
-a|
-Member-Value
-
-a|
-Meaning
-
-a|
-Default
-
-a|
-operationName
-
-a|
-Name of the wsdl:operation matching this method.
+|Member-Value
+|Meaning
+|Default
 
-a|
-Name of the Java method
+|operationName
+|Name of the wsdl:operation matching this method.
+|Name of the Java method
 
-a|
-action
-
-a|
-The action for this operation. For SOAP bindings, this determines the
+|action
+|The action for this operation. For SOAP bindings, this determines the
 value of the soap action.
+|""
 
-a|
-""
-
-a|
-exclude
-
-a|
-Marks a method to NOT be exposed as a web method. Used to stop an
+|exclude
+|Marks a method to NOT be exposed as a web method. Used to stop an
 inherited method from being exposed as part of this web service.
 
 If this element is specified, other elements MUST NOT be specified for
 the @WebMethod.
-
-a|
-False
-
+|False
 |===
 
 This member-value is not allowed on endpoint interfaces.
```

Comment:
this sentence belongs to the meaning column of the @WebMethod.exclude (same as in 4.2.1 for @WebService. endpointInterface)

Reply:
ok, resolved by https://github.com/eclipse-ee4j/jws-api/pull/69/commits/3a180473a64e0263e304fb1ccef0dd91acc070ae